### ISSUE TITLE
Training and inference with "co-detr-vit-large-coco-instance"

### ISSUE THREAD

**Issue by Shengcao-Cao** [Sense-X/Co-DETR#183]
Hi,

Thanks for sharing the great work with the community!

I'm very interested in reproducing and evaluating your latest model with ViT-L backbone for instance segmentation (https://huggingface.co/zongzhuofan/co-detr-vit-large-coco-instance). However, I'm having some questions and would like to request your help:

1. Configuration file: There does not seem to be a config file for COCO instance segmentation. I try to create one by adding the differences between [co_dino_5scale_lsj_vit_large_lvis_instance](https://github.com/Sense-X/Co-DETR/blob/main/projects/configs/co_dino_vit/co_dino_5scale_lsj_vit_large_lvis_instance.py) vs. [co_dino_5scale_lsj_vit_large_lvis](https://github.com/Sense-X/Co-DETR/blob/main/projects/configs/co_dino_vit/co_dino_5scale_lsj_vit_large_lvis.py) into [co_dino_5scale_vit_large_coco](https://github.com/Sense-X/Co-DETR/blob/main/projects/configs/co_dino_vit/co_dino_5scale_vit_large_coco.py), but I'm not sure if this is the best way to do so. It would be great if you could provide one config file for co_dino_5scale_vit_large_coco_instance in projects/configs/co_dino_vit.
2. Objects365 pretraining: I suppose this model is finetuned on COCO after training on Objects365. Would you mind also sharing the Objects365-pretrained model weights, so that I can directly start from there and train on COCO?
3. Number of GPUs: How many GPUs in total did you use to train this model?

Thank you very much for your time and consideration.

Best,
Shengcao

**Comment by TempleX98** [comment]
1. Please check the updated config of ViT: https://github.com/Sense-X/Co-DETR/blob/main/projects/configs/co_dino_vit/co_dino_5scale_vit_large_coco_instance.py
2. I will upload this model to Hugging Face as soon as possible.
3. We use 64 A100 80GB GPUs for finetuning. Using a smaller learning rate with a smaller batch size is okay if you have fewer GPUs.

**Comment by TempleX98** [comment]
1. It's the O365 pretrained model.
2. Yes, I added the learning rate layerwise decay just now. Please update your code to the latest commit. The learning rate should be linearly scaled based on the actual batch size before training.

**Comment by Shengcao-Cao** [comment]
Thank you very much for the updates!
